app/users/resources.py

```diff
@@ -1,25 +1,13 @@
 import json
 
 from flask import request
-from flask_restful import Resource, reqparse
+from flask_restful import Resource
 
 from app import db, User
-from app.lib.response_util import buildOkResponse
+from app.lib.response_util import buildOkResponse, _returnUser
```

Comment:
Let's import the whole `response_util` module rather than the individual functions.